brvs-rules/src/main/java/iroha/validation/rules/impl/billing/BillingRule.java

```diff
@@ -53,13 +53,14 @@
 public class BillingRule implements Rule {
 
   private static final Logger logger = LoggerFactory.getLogger(BillingRule.class);
+  private static final String BTC_ASSET = "btc#bitcoin";
+  private static final String WITHDRAWAL_DESCRIPTION = "f842c3e1858df8c85794efacd06871f806afab04e0458c44488d2ffb3f071075";
```

Comment:
name is confusing, mb WITHRDAWAL_FEE_DESCRIPTION?

Reply:
Okay, done